site/src/pages/UserSettingsPage/SSHKeysPage/SSHKeysPage.tsx

```diff
@@ -19,12 +20,13 @@ export const Language = {
     "You will need to replace the public SSH key on services you use it with, and you'll need to rebuild existing workspaces.",
   confirmLabel: "Confirm",
   cancelLabel: "Cancel",
+  errorRegenerateSSHKey: "Error on regenerate the SSH Key",
```

Comment:
Just taking this opportunity to fix a typo, I know it was like this before.
```suggestion
  errorRegenerateSSHKey: "Error on regenerating the SSH Key",
```